pkg/apis/kudo/v1beta1/instance_validator.go

```diff
@@ -0,0 +1,36 @@
+package v1beta1
+
+import (
+	"fmt"
+
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+
+	"github.com/kudobuilder/kudo/pkg/util/kudo"
+
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+var _ kudo.Validator = &Instance{}
```

Comment:
I'm not sure this is necessary... I suppose it prevents future refactors which remove or change the methods defined below with a compilation error.

kind of clever... is there another purpose?

adding to ken's bag of tricks... thx!

Reply:
I actually took this from controller-runtime example :) I can remove it if people feel like it shouldn't be here but as you said, it's pretty neat trick :)

https://github.com/kubernetes-sigs/controller-runtime/blob/master/examples/crd/pkg/resource.go#L66